array_api_compat/common/_helpers.py

```diff
@@ -53,7 +53,7 @@ def _check_api_version(api_version):
     if api_version is not None and api_version != '2021.12':
         raise ValueError("Only the 2021.12 version of the array API specification is currently supported")
 
-def array_namespace(*xs, api_version=None, _use_compat=True):
+def array_namespace(*xs, api_version=None, _use_compat=True, fallback_namespace=None):
```

Comment:
The fallback namespace should still be an array API compatible namespace. So if someone does `fallback_namespace=numpy` instead of `fallback_namespace=array_api_compat.numpy`, we might want to automatically convert for them.

Reply:
I added some logic to try to convert the namespace. I did not see a function to do that but I suppose this could be made into it's own function if there are more needs.